dojo/tools/cyclonedx/parser.py

```diff
@@ -329,6 +329,14 @@ def _get_findings_json(self, file, test):
                 for reference in vulnerability.get("references", []):
                     if re.fullmatch("CVE-[0-9]+-[0-9]+", reference.get("id", "")):
                         finding.cve = reference.get("id")
+                # check if the tool made the CVE as an id
```

Comment:
CVE and references are now 2 times in the JSON parser. CVE is directly before that in lines 328 - 331. References are filled from advisories in lines 292 - 299

Reply:
it is not a problem. I saw the tools handle the CVE differently. so we are able to get the CVE if it's a reference **or** if it's the id of the vulnerability. many parsers do that. in the end we extract successfully the CVE

Reply:
Thats alright for the CVE. But lines 335 - 339 do the same what has been done in lines 290 - 299. I would like to keep 290 - 299, because it would not bring None values in the references if a field is not used and it keeps the word Title for the title attribute.

Reply:
fixed